earth_enterprise/src/server/wsgi/serve/push/search/util/search_schema_parser.py

```diff
@@ -394,7 +412,8 @@ def main(argv):
   parser = SearchSchemaParser(db_updater)
 
   start = time.time()
-  (num_fields, sql_search, balloon_style) = parser.Parse(poifile, "test_poi")
+  # TODO(RAW): add prefix path to test this
```

Comment:
Still a TODO here.

Reply:
Oh yes.  This looks like a unit test for this module.  Do the unit tests for python get run differently than the unit tests for C++?  I noticed that python .pyc files are not created when I run 'scons build' so I'm guessing the tests are run differently too.

Reply:
I'm not sure there are python unit tests for GEE server yet. I don't want to overload this PR with building a unit testing framework, so it's OK with me if we skip python testing for now.